instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo/config.go

```diff
@@ -5,16 +5,22 @@ package otelmongo // import "go.opentelemetry.io/contrib/instrumentation/go.mong
 
 import (
 	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
 	"go.opentelemetry.io/otel/trace"
 )
 
 // ScopeName is the instrumentation scope name.
 const ScopeName = "go.opentelemetry.io/contrib/instrumentation/go.mongodb.org/mongo-driver/v2/mongo/otelmongo"
 
 // config is used to configure the mongo tracer.
+// when you want to disable metrics or traces, use the [WithMeterProvider] options
+// to set a [go.opentelemetry.io/otel/metric/noop.MeterProvider] or
+// [WithTracerProvider] options to set a [go.opentelemetry.io/otel/trace/noop.TracerProvider]
```

Comment:
[docs] This comment seems unnecessary given that config is an un-exported symbol. See suggestion for WithMetricProvider for deferring the comment.